### DISCUSSION TITLE
Need help with package intallation

### DISCUSSION THREAD

**Discussion by zeinebCh** [lava-nc/lava-dl#160] [General]
Hello guys,
I am trying to install the lava-dl package. So I activated my virtual environment. And then I run the following command:
 pip install --no-cache-dir lava-dl
and I got the following error:
ERROR: Could not find a version that satisfies the requirement lava-dl (from versions: none)
ERROR: No matching distribution found for lava-dl

Could you please help me to solve this problem?
Thank you

**Comment by bamsumit** [comment]
Hi @zeinebCh, please follow the instructions here: https://github.com/lava-nc/lava-dl#installation
